### DISCUSSION TITLE
Flatpak: How to install new dicitonaries for Spell Checking?

### DISCUSSION THREAD

**Discussion by PrSunflower** [KDE/ghostwriter#835] [Q&A] [answered]
Hello, I am running _ghostwriter_ 2.2.0 on Linux.

In `Settings > Preferences > Spell check` I have a lot of **English** languages but I would like to have **French** spell checking.

I have just gone through the entire website and the new KDE GitHub page but I didn't find any information on this.

Could someone please tell me how to install a new dictionary?

**Comment by wereturtle** [comment]
I’m not sure of the exact package name for your OS but install the hunspell French dictionary. Just do a search for “hunspell” with your package manager. It will probably be named something like “hunspell-fr”. 

-- 
  Megan
  ghostwriter developer

**Reply by PrSunflower** [reply]
Thank you :slightly_smiling_face: 

May you please add these instructions on the website or the README.md? I would think that Spell Checking might be an important feature for users.

Thank you, wishing you a nice day. :star2:

**Reply by PrSunflower** [reply]
Whoops sorry @wereturtle , it's not working. I forgot to say that I am running the Flatpak version.

Will my Hunspell dictionaries work if they come from APT?

```
user@PC:~$ apt list --installed *hunsp*
Listing... Done
hunspell-en-us/byzantium,now 1:2019.10.06-1 all [installed,automatic]
hunspell-fr-classical/byzantium,now 1:7.0-1 all [installed,automatic]
hunspell-fr/byzantium,now 1:7.0-1 all [installed]
libhunspell-1.7-0/byzantium,now 1.7.0-3 amd64 [installed,automatic]
```

**Comment by wereturtle** [comment, marked as answer]
Due to Flatpak’s sandboxing they very wrong might not work. You could try contacting the Flatpak maintainers. (No longer me.) They might know of a workaround. 

-- 
  Megan
  ghostwriter developer


On Sat, Nov 5, 2022, at 1:37 PM, Pr. Sunflower wrote:
> 
> 
> Whoops sorry @wereturtle <https://github.com/wereturtle> , it's not working. I forgot to say that I am running the Flatpak version.
> 
> Will my Hunspell dictionaries work if they come from APT?
> 
> ***@***.***:~$ apt list --installed *hunsp*
> Listing... Done
> hunspell-en-us/byzantium,now 1:2019.10.06-1 all [installed,automatic]
> hunspell-fr-classical/byzantium,now 1:7.0-1 all [installed,automatic]
> hunspell-fr/byzantium,now 1:7.0-1 all [installed]
> libhunspell-1.7-0/byzantium,now 1.7.0-3 amd64 [installed,automatic]
`

**Reply by PrSunflower** [reply]
Oh well... 😅

Thank you for your answer, I'll see what I can do :slightly_smiling_face:
